app/presenters/two_factor_options_presenter.rb

```diff
@@ -56,6 +65,10 @@ def intro
     end
   end
 
+  def show_cancel_aal2plus
```

Comment:
As with the methods that it uses in the implementation, if this returns a boolean, I'd [conventionally](https://rubystyle.guide/#bool-methods-qmark) expect it to be suffixed with `?`.

Also, my fault for naming the ticket, but I don't know that the name "aal2 plus" has common usage to be widely understood. (vs. how it's called here as a union of the concepts "phishing resistant only" and "piv cac required")

I wonder if we could do one of the following:

- Name it more semantically to as far as what it's controlling (e.g. `show_cancel_return_to_sp?`)
- Incorporate the handling into `skip_path` and `skip_label` so that we don't need a separate branch path in the view

Reply:
I changed the method label, and added the ? - but I like what you're saying in the second bullet and will take a look at implementing that.